src/patternfly/components/FormControl/examples/FormControl.md

```diff
@@ -174,10 +174,10 @@ Resizes in both directions
 | -- | -- | -- |
 | `.pf-v5-c-form-control` | `<div>` |  Initiates a container for an input, textarea or select. For styling of checkboxes or radios see the [checkbox component](/components/checkbox) or [radio component](/components/radio). **Required**  |
 | `.pf-v5-c-form-control__utilities` | `<div>` |  Initiates a container for elements like icons to be associated with the form control.  |
-| `.pf-v5-c-form-control__icon` | `<div>` |  Creates a container for an icon associated with a text input.  |
+| `.pf-v5-c-form-control__icon` | `<div>` |  Creates a container for an icon associated with a form control.  |
 | `.pf-v5-c-form-control__toggle-icon` | `<div>` |  Initiates a toggle icon for a form select.  |
-| `.pf-m-resize-vertical` | `.pf-m-form-control` | Modifies a `pf-v5-c-form-control` element containing a text area so it can only be resized vertically. |
-| `.pf-m-resize-horizontal` | `.pf-m-form-control` | Modifies a `.pf-v5-c-form-control` element containing a textarea so it can only be resized horizontally. |
+| `.pf-m-resize-vertical` | `.pf-m-form-control` | Modifies a form control element containing a text area so it can only be resized vertically. |
```

Comment:
```suggestion
| `.pf-m-resize-vertical` | `.pf-v5-c-form-control` | Modifies a form control element containing a textarea so it can only be resized vertically. |
```